scripts/confidence_ensembles/build_ensemble.py

```diff
@@ -156,32 +295,84 @@ def cleanup_subsampled_manifests(subsampled_manifests):
         os.remove(manifest)
 
 
+def compute_all_confidences(transcription, tune_confidence_cfg: TuneConfidenceConfig) -> Dict[ConfidenceSpec, float]:
+    conf_values = {}
+
+    for exclude_blank in tune_confidence_cfg.exclude_blank:
+        filtered_logprobs = get_filtered_logprobs(transcription, exclude_blank)
+        vocab_size = filtered_logprobs.shape[1]
+        for aggregation in tune_confidence_cfg.aggregation:
+            aggr_func = get_confidence_aggregation_bank()[aggregation]
+            for conf_type in tune_confidence_cfg.confidence_type:
+                conf_func = get_confidence_measure_bank()[conf_type]
+                if conf_type == "max_prob":  # skipping alpha in this case
+                    conf_value = aggr_func(conf_func(filtered_logprobs, v=vocab_size, t=1.0)).cpu().item()
+                    conf_values[ConfidenceSpec(exclude_blank, aggregation, conf_type, 1.0)] = conf_value
```

Comment:
Split into separate lines

Reply:
Can you please clarify what exactly should be split here?

Reply:
Oh it's my mistake. You're using dataclass as key. Just looks weird but works I guess